example/example.cpp

```diff
@@ -26,9 +26,32 @@ void custom_flags_example();
 #include "spdlog/spdlog.h"
 #include "spdlog/cfg/env.h"  // support for loading levels from the environment variable
 #include "spdlog/fmt/ostr.h" // support for user defined types
+#include "spdlog/common.h"
+
+spdlog::file_event_handlers_t file_event_handlers;
```

Comment:
no need to make it global

Reply:
file_event_handlers is referenced by other example functions.

Reply:
I prefer not to change the original example functions and keep it simple. Lets just have single function in the end that shows how to use it.

Reply:
ok.
I have changed that. Just pass file_event_handlers to the rotating_logger